widgets-bundle/getting-started/creating-a-widget.md

```diff
@@ -108,18 +77,20 @@ function __construct() {
 		array(
 			'text' => array(
 				'type' => 'text',
-				'label' => __('Hello world! goes here.', 'siteorigin-widgets'),
-				'default' => 'Hello world!'
+				'label' => __( 'Hello world! goes here.', 'hello-world-widget-text-domain' ),
+				'default' => 'Hello world!',
 			),
 		),
 
 		//The $base_folder path string.
-		plugin_dir_path(__FILE__)
+		plugin_dir_path( __FILE__ )
 	);
 }
+siteorigin_widget_register( 'hello-world-widget', __FILE__, 'Hello_World_Widget' );
+
 ```
 
-Once you have your constructor implemented like the above example, you should see your widget's name and description being displayed in the various widget configuration lists. The Hello World widget will now also display a text field in the Edit Widget form containing the text 'Hello world!', which can be edited and saved.
+Once you have implemented your widget like the above example, your widget will be listed in the SiteOrigin Widgets List. This list can be accessed by navigating to **Plugins > SiteOrigin Widgets**. In the above example, the Hello World widget will be listed. This widget will contain a text field in the Edit Widget form containing the text 'Hello world!', which can be edited and saved. It won't however output the text on the frontend without a Widget Template.
```

Comment:
Should we go with lower case "L" for list?

For the last sentence a small change adding commas:

` won't, however, output the text on the frontend without a Widget Template.`